frontend/src/__tests__/cypress/cypress/tests/mocked/home/homeAdmin.cy.ts

```diff
@@ -84,34 +90,33 @@ describe('Home page Admin section', () => {
     });
 
     homePage.visit();
-
-    cy.get('#dashboard-page-main').find('[class="pf-v5-c-page__main"]').scrollTo('bottom');
-    cy.findByTestId('landing-page-admin').should('not.exist');
+    homePage.scrollToBottom();
```

Comment:
Unnecessary to scroll if we are asserting `not.exist`. Only visibility checks requires the element to be visible in the viewport.